source/helpers/name-pattern-validator.js

```diff
@@ -17,10 +18,11 @@ import {
  */
 
 export default (namePattern) => {
-  if (new RegExp(`${IMAGE_NAME_FORMAT_ORIGIN}|${IMAGE_NAME_FORMAT_HASH}|${IMAGE_NAME_FORMAT_DATE}|false`)
+  if (new RegExp(`${IMAGE_NAME_FORMAT_ORIGIN}|${IMAGE_NAME_FORMAT_HASH}|${IMAGE_NAME_FORMAT_DATE}`)
```

Comment:
If the purpose of this RegExp test is just to find if `namePattern` is one of three static words: `origin`, `hash`, `date`, then using `Array`'s `includes` is enough. RegExp calculation is way more expensive.